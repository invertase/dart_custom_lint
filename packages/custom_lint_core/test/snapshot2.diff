Message: `Another`
Priority: 1
Diff for file `file.dart:2`:
```
void main() {
-   print('Hello world');
+   Helloprint('Hello world');
}
```
---
