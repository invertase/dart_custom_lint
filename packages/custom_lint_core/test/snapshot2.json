[
  {
    "result": "void main() {\n  Helloprint('Hello world');\n}\n",
    "changes": [
      {
        "priority": 1,
        "change": {
          "message": "Another",
          "edits": [
            {
              "fileStamp": 0,
              "edits": [
                {
                  "offset": 16,
                  "length": 0,
                  "replacement": "Hello"
                }
              ]
            }
          ],
          "linkedEditGroups": []
        }
      }
    ]
  }
]