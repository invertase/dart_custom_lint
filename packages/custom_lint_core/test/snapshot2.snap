Message: `Another`
<<<< start: 16 -- end: 16
==== start: 16 -- end: 21
Hello
>>>>
